construct.yaml

```diff
@@ -35,9 +36,9 @@ exclude:
 packages:
 #  - python-2.7.9-0.tar.bz2
 
-#install_in_dependency_order: True
+install_in_dependency_order: True
 
-keep_pkgs: True
+# keep_pkgs: True
```

Comment:
why?

Reply:
It takes up extra space on the user's computer, the default is false, and the installer works without this flag.